### PR TITLE
SONAR-7580 only sonar.embeddedDatabase.port required to set H2 port

### PR DESCRIPTION
**Pull request by sns-seb**
in addition, if sonar.jdbc.url is set, a warning will be logged

### CODE REVIEW

File: sonar-application/src/main/java/org/sonar/application/JdbcSettings.java [hunk 1]
```diff
@@ -91,6 +119,27 @@ void checkUrlParameters(Provider provider, String url) {
     }
   }
 
+  private static void warnIfUrlIsSet(String port, String existing, String expectedUrl) {
+    if (isNotEmpty(existing)) {
+      Logger logger = LoggerFactory.getLogger(JdbcSettings.class);
```
Comment (SonarTech) [inline sonar-application/src/main/java/org/sonar/application/JdbcSettings.java]: MAJOR Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1312)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by sns-seb** [comment]
@simonbrandhof  back to you, PR is fixed

**Comment by simonbrandhof** [comment]
LGTM

**Pull request merged**